src/clients/c++/request_http.h

```diff
@@ -139,6 +139,25 @@ class ProfileHttpContext {
       const std::map<std::string, std::string>& headers, bool verbose = false);
 };
 
+//==============================================================================
+/// ControlHttpContext is the HTTP instantiation of ControlContext.
+///
+class ControlHttpContext {
+ public:
+  /// Create context that controls model ready state on a server using HTTP
```

Comment:
"controls mode ready state" is a strange way to say it. Can you just directly say that it allows models to be loaded and unloaded.